backends/p4tools/modules/testgen/targets/bmv2/test/gtest_utils.h

```diff
@@ -0,0 +1,21 @@
+#ifndef BACKENDS_P4TOOLS_MODULES_TESTGEN_TARGETS_BMV2_TEST_GTEST_UTILS_H_
+#define BACKENDS_P4TOOLS_MODULES_TESTGEN_TARGETS_BMV2_TEST_GTEST_UTILS_H_
+
+#include "backends/p4tools/modules/testgen/test/gtest_utils.h"
+#include "backends/p4tools/modules/testgen/test/small-step/util.h"
+
+namespace P4Tools::Test {
+
+/// Sets up the correct context for a P4Testgen BMv2 test.
+class P4TestgenBmv2Test : public P4TestgenTest {};
+
+/// Creates a test case with the @hdrFields for stepping on an @expr.
+std::optional<const P4ToolsTestCase> createSmallStepExprTest(const std::string &hdrFields,
+                                                             const std::string &expr);
```

Comment:
Is this BMv2 specific, or generic? The other things in this file mention BMv2, but this one does not, but apparently it is still BMv2 specific. So maybe just add BMv2 to the test function name? (since this is not in any BMv2 namespace).

Reply:
It is BMv2-specific because it creates a V1Model program, fixed.